airbyte-integrations/connectors/source-iterable/source_iterable/utils.py

```diff
@@ -0,0 +1,20 @@
+#
+# Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+#
+
+import dateutil.parser
+import pendulum
+
+
+def dateutil_parse(text):
```

Comment:
could you please unit-test memory usage of this function?

Reply:
if you are talking about checking memory diff with

```python
tracemalloc.get_traced_memory()
```

This does not work, I am not sure why, maybe because memory leak in C level which is not counted by `tracemalloc`

Reply:
Of course I am talking about `pendulum.parse` which leak memory